api/src/main/resources/liquibase.xml

```diff
@@ -64,4 +64,13 @@
             UPDATE global_property SET property_value = '1732AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' WHERE property='order.durationUnitsConceptUuid';
         </sql>
     </changeSet>
+
+    <changeSet id="ugandaemrpoc-14-08-2020-04" author="dbaluku">
+        <comment>Updating old data of concept id 90253 to 165275 as family member
```

Comment:
Migrate data from concept id 90253 (HIV Status) to 165275 (Family Member HIV Status) unknown answer 90001 to 165165 in the on the ART Summary Page Family Tracking Section from 2.x to 3.x